lib/Doctrine/DBAL/Platforms/MySqlPlatform.php

```diff
@@ -654,7 +654,7 @@ protected function getPreAlterTableIndexForeignKeySQL(TableDiff $diff)
                         $indexClause = 'UNIQUE INDEX ' . $addIndex->getName();
                     }
 
-                    $query = 'ALTER TABLE ' . $table . ' DROP INDEX ' . $remIndex->getName() . ', ';
+                    $query = 'ALTER TABLE ' . $table . ' DROP INDEX `' . $remIndex->getName() . '`, ';
```

Comment:
The quoting of an identifier (any identifier) should be performed by using `$this->quoteStringLiteral()`, and not by hardcoding backticks